action/membership/routes.go

```diff
@@ -7,8 +7,8 @@ type membership struct {
 	PlanID uint `json:"plan_id" validate:"required"`
 }
 
-// Router - Group of membership router
-func Router() chi.Router {
+// UserRouter - Group of membership router
+func UserRouter() chi.Router {
 	r := chi.NewRouter()
 
 	r.Get("/", list)    // GET /memberships - return list of memberships
```

Comment:
UserRouter must access *GET /memberships/{membership_id}* only

Reply:
user should be able to create membership also